semgrep/semgrep/output.py

```diff
@@ -329,7 +324,7 @@ def output(
             else:
                 if output:
                     try:
-                        print(output, file=self.stdout)
```

Comment:
@underyx This was causing findings to not appear in test_ci stdout (it was bypassing CliRunner.invoke's stdout capture). We were checking findings with the network capture part of the test but thought would be nice to have test for --json output and regular output with this PR. Looks like we never use the stdout stderr args of OutputHandler anyway so I think this is fine